matrix_client/client.py

```diff
@@ -379,3 +379,27 @@ def get_user(self, user_id):
         """
 
         return User(self.api, user_id)
+
+    def get_public_rooms_ids(self):
```

Comment:
Is there an especially good reason for having this over ``get_public_rooms`` and doing the operation inside your own code?